pom.xml

```diff
@@ -352,6 +358,16 @@
                 <version>${spark.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.antlr</groupId>
+                <artifactId>antlr4-runtime</artifactId>
```

Comment:
Is this coming from hadoop libraries ? You can run `mvn dependency:tree -Dincludes ` to find out which one's conflicting.

Reply:
For now i think the simplest solution is moving this to the antlr dep to the rest-service pom.xml and using `dependency management` to ensure this picks the correct version. This should leave all other modules untouched and resolve the dep conflict for the service.